leo/commands/prove.rs

```diff
@@ -14,73 +14,64 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{cli::*, cli_types::*, commands::SetupCommand, errors::CLIError};
-use leo_package::{outputs::ProofFile, root::Manifest};
+use super::setup::Setup;
+use crate::{commands::Command, context::Context};
+use anyhow::Result;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates